http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200501_kubernetes_tables.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="jcanas" id="kubernetes_cluster_table">
+        <createTable tableName="KUBERNETES_CLUSTER">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="googleProject" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="clusterName" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="location" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="VARCHAR(254)" defaultValue="STATUS_UNSPECIFIED">
+                <constraints nullable="false"/>
+            </column>
+            <column name="serviceAccountInfo" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="samResourceId" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="creator" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="createdDate" type="TIMESTAMP(6)" defaultValue="0000-00-00 00:00:00.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="destroyedDate" type="TIMESTAMP(6)" defaultValue="1970-01-01 00:00:01.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="dateAccessed" type="TIMESTAMP(6)" defaultValue="0000-00-00 00:00:00.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="apiServerIp" type="VARCHAR(254)"/>
+            <column name="networkName" type="VARCHAR(254)"/>
+            <column name="subNetworkName" type="VARCHAR(254)"/>
+            <column name="subNetworkIpRange" type="VARCHAR(254)"/>
+        </createTable>
+        <createIndex indexName="IDX_KUBERNETES_CLUSTER_ID" tableName="KUBERNETES_CLUSTER">
```

Comment:
This index may not be needed since it's already the primary key

Reply:
huh, I didn't know primary keys were automatically indexed in SQL, TIL